src/controllers/referralHelper.js

```diff
@@ -60,7 +67,7 @@ exports.getReferral = async (usersId, email) => {
  * @param {dict} response
  */
 async function generateReferral(usersId, email, response) {
-  const referralCode = generateReferralCode();
+  const referralCode = generateReferralCode(6);
```

Comment:
it will be better to make `6` as a constant, like the error messages.

Reply:
added constants